plugin/src/test/java/org/opensearch/ml/rest/RestMLRemoteInferenceIT.java

```diff
@@ -435,17 +430,19 @@ public void testOpenAITextEmbeddingModel() throws IOException, InterruptedExcept
         responseMap = parseResponseToMap(response);
         taskId = (String) responseMap.get("task_id");
         waitForTask(taskId, MLTaskState.COMPLETED);
-        String predictInput = "{\n" + "  \"parameters\": {\n" + "      \"input\": \"The food was delicious\"\n" + "  }\n" + "}";
+        String predictInput = "{\n"
```

Comment:
Can we have another test with another charset? Maybe with ASCII or something else?